streaming/src/main/scala/org/apache/spark/streaming/receiver/RateLimiter.scala

```diff
@@ -61,4 +62,21 @@ private[receiver] abstract class RateLimiter(conf: SparkConf) extends Logging {
         rateLimiter.setRate(newRate)
       }
     }
+
+  /**
+   * Get the initial rateLimit to initial rateLimiter
+   * @return
+   */
+  def getInitialRateLimit() : Long = {
+    if (RateController.isBackPressureEnabled(conf)) {
+      val initialRate = conf.getLong("spark.streaming.receiver.initialRate", 1000)
```

Comment:
This should be `spark.streaming.backpressure.initialRate` to keep it in the same name space as "spark.streaming.backpressure.enabled"